diagnostic.md

```diff
@@ -150,7 +175,8 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+On one lione, in order from left to starting with divOne ... see below
+1 2 3 4 5
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5